docs/binaries/santad.md

```diff
@@ -4,42 +4,58 @@ parent: Binaries
 
 # santad
 
-The santad process does the heavy lifting when it comes to making decisions
-about binary executions. It also handles brokering all of the XPC connections
-between the various components of Santa. It does all of this with performance
-being at the forefront.
+The `santad` process does the heavy lifting when it comes to making decisions
```

Comment:
Probably worth mentioning here that while we still refer to it as santad, it's more typically seen on the system as com.google.santa.daemon.

Reply:
Added a note at the top of the doc.